plugins/arDominionB5Plugin/templates/_layout_start_webpack.php

```diff
@@ -9,7 +9,7 @@
     <%= htmlWebpackPlugin.tags.headTags %>
     <?php echo get_component_slot('css'); ?>
   </head>
-  <body class="d-flex flex-column min-vh-100 <?php echo $sf_context->getModuleName(); ?> <?php echo $sf_context->getActionName(); ?>">
+  <body class="d-flex flex-column min-vh-100 <?php echo $sf_context->getModuleName(); ?> <?php echo $sf_context->getActionName(); ?><?php echo sfConfig::get('app_show_tooltips') ? ' show-edit-tooltips' : ''; ?>">
```

Comment:
Thinking about this again, while I still think is the best solution for us, we should take a note for custom theme upgrades, as it will be needed in all B5 themes _layout_start_webpack.php file.